src/dev/LCD.cpp

```diff
@@ -0,0 +1,95 @@
+    this->commWrite(0xB0 + page);   //writes the page address (4 bits, 8 rows selcted by values 0-7 )
+    this->commWrite(0x10 + col_up); //writes the first 4 bits of the column select (out of 8 bits)
+    this->commWrite(0x00 + col_low);//writes the second 4 bits of the column select (out)
+
+    this->dataWrite(data);//writes 8 vertical bits based on value between 0-255 based on bits set ex: 01001100(0x4C) is |WHITE|
+                          //                                                                                                 |BLACK|
+                          //                                                                                                 |WHITE|
+                          //                                                                                                 |WHITE|
+                          //                                                                                                 |BLACK|
+                          //                                                                                                 |BLACK|
+                          //                                                                                                 |WHITE|
+                          //                                                                                                 |WHITE|
+}
+
+void LCD::clearLCD(unsigned char* bitMap) {
+    unsigned int i, j;
+    unsigned char page = 0xB0;
+    this->commWrite(0xAE);         //Display OFF
+    this->commWrite(0x40);         //Display start address + 0x40
+    for (i = 0; i < 8; i++) {      //64 pixel display / 8 pixels per page = 8 pages
+        this->commWrite(page);     //send page address
+        this->commWrite(0x10);     //column address upper 4 bits + 0x10
+        this->commWrite(0x00);     //column address lower 4 bits + 0x00
+        for (j = 0; j < 128; j++) {//128 columns wide
+            this->dataWrite(0x00); //write clear pixels
+            bitMap++;
+        }
+        page++;//after 128 columns, go to next page
+    }
+    this->commWrite(0xAF);
+}
+
+void LCD::displayMap(unsigned char* bitMap) {
+    unsigned int i, j;
+    unsigned char page = 0xB0;
+    this->commWrite(0xAE);           //Display OFF
+    this->commWrite(0x40);           //Display start address + 0x40
+    for (i = 0; i < 8; i++) {        //64 pixel display / 8 pixels per page = 8 pages
+        this->commWrite(page);       //send page address
+        this->commWrite(0x10);       //column address upper 4 bits + 0x10
+        this->commWrite(0x00);       //column address lower 4 bits + 0x00
+        for (j = 0; j < 128; j++) {  //128 columns wide
+            this->dataWrite(*bitMap);//write clear pixels
+            bitMap++;
+        }
+        page++;//after 128 columns, go to next page
+    }
+    this->commWrite(0xAF);
+}
+
+void LCD::initLCD() {
+
+    this->reset.writePin(EVT::core::IO::GPIO::State::HIGH);
```

Comment:
This pin is written high here, but only ever written low in the constructor. We should check to understand why that is